src/core/search/ast_expr.h

```
@@ -45,6 +45,13 @@ struct AstRangeNode {
   double lo, hi;
 };
 
+struct AstGeoNode {
+  AstGeoNode(double lat, double lon, size_t radius, std::string unit);
+  double lat, lon;
+  size_t radius;
+  std::string unit;
+};
```

TODO: parse unit from lexer (so we can throw the error in time)